libkbfs/block_journal.go

```diff
@@ -115,6 +118,8 @@ type blockJournalEntry struct {
 	// Must have exactly one entry with one context for blockPutOp
 	// and addRefOp.
 	Contexts map[BlockID][]BlockContext
+	// Only used for mdRevMarkerOps.
+	Revision MetadataRevision
```

Comment:
May as well mark this and `Contexts` as `omitempty`. Also, please amend the comment for `Contexts` to mention that it's used for all ops but `mdRevMarkerOps`.